lib/recurly/fraud.js

```diff
@@ -99,9 +99,17 @@ export class Fraud extends Emitter {
             initializerElement.setAttribute('data-event', 'load');
 
             const form = dom.element(configuredForm) || this.getHostedFieldParentForm();
-            form.appendChild(sessionIdInputElement);
-            form.appendChild(scriptElement);
-            form.appendChild(initializerElement);
+            if (form) {
+              const nodes = [
+                sessionIdInputElement,
+                scriptElement,
+                initializerElement
+              ];
+
+              nodes.forEach(node => form.appendChild(node));
+              this.attachedNodes = nodes;
+            }
```

Comment:
This is the essential fix for the fraud form detection. If an impl lacked a `form` param or did not have a valid `hostedFields` instance, this would have errored on `appendChild`.

Reply:
This also assigns `attachedNodes` to restore the functionality of `destroy`.